src/python/pants/backend/docker/target_types.py

```diff
@@ -311,14 +312,26 @@ def options(self, value_formatter: OptionValueFormatter) -> Iterator[str]:
             )
 
 
+class DockerBuildKitOptionField:
+    """Mixin to indicate a BuildKit-specific option."""
+
+    @abstractmethod
+    def options(self, value_formatter: OptionValueFormatter) -> Iterator[str]:
```

Comment:
I'm not sure if this is the _best_ way to do this, but I needed to add it to fix a mypy error